README.md

```diff
@@ -33,9 +46,13 @@ make deps
 make run
 ```
 
-## Local (Development)
+## Docker installation and usage
+
+### Requirements
+
+*   Install docker engine (<https://docs.docker.com/install/>)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[unordered-list-marker-style] Marker style should be `-`](https://app.codacy.com/app/joaoaneto/knot-babeltower/pullRequest?prid=4651373)